src/qt/optionsmodel.cpp

```diff
@@ -260,6 +275,23 @@ bool OptionsModel::getCoinControlFeatures()
     return fCoinControlFeatures;
 }
 
+bool OptionsModel::getLimitTxnDisplay()
+{
+    return fLimitTxnDisplay;
+}
+
+QDate OptionsModel::getLimitTxnDate()
+{
+    return limitTxnDate;
+}
+
+int64_t OptionsModel::getLimitTxnDateTime()
+{
+    QDateTime limitTxnDateTime(limitTxnDate);
+
+    return static_cast<int64_t>(limitTxnDateTime.toTime_t());
```

Comment:
```suggestion
    return limitTxnDate.startOfDay().toSecsSinceEpoch();
```

`QDateTime::toTime_t()` is deprecated. Noticed this while doing the voting stuff.

Reply:
Thanks!

Reply:
startOfDay() is not is only available in >= 5.14

Reply:
I think this will work...

int64_t OptionsModel::getLimitTxnDateTime()
{
    QDateTime limitTxnDateTime(limitTxnDate);

    return limitTxnDateTime.toSecsSinceEpoch();
}

Reply:
Ah, bummer. Yours looks good to me.